editor/editor_file_system.h

```diff
@@ -69,6 +71,8 @@ class EditorFileSystemDirectory : public Object {
 		String script_class_extends;
 		String script_class_icon_path;
 		String icon_path;
+		bool link = false;
```

Comment:
I'm wondering if using only a String should be easier. If it's empty, it's not a link. That way you don't have to call two method in the EditorFileSystem to check if a folder or a file is a link. Also, it prevent the potentiel problem that is present now if a link if changed to a real file, the bool is updated but the `link_path` still contains the old link path.
Seriously, it's not really a problem, more a suggestion.